pkg/util/system/go.mod

```diff
@@ -0,0 +1,32 @@
+module github.com/DataDog/datadog-agent/pkg/util/system
+
+go 1.21
+
+require (
+	github.com/DataDog/datadog-agent/pkg/util/filesystem v0.50.2
+	github.com/DataDog/datadog-agent/pkg/util/log v0.50.2
+	github.com/DataDog/datadog-agent/pkg/util/pointer v0.50.2
+	github.com/DataDog/datadog-agent/pkg/util/testutil v0.50.2
+	github.com/DataDog/datadog-agent/pkg/util/winutil v0.36.1
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏I think you need to add `replace` directives for those imports (+ `pkg/util/scrubber` which is indirect)